RadixWallet/Features/SecurityCenterFeature/SecurityFactors/SecurityFactors+View.swift

```diff
@@ -54,50 +37,38 @@ private extension SecurityFactors.View {
 		}
 	}
 
-	func rows(viewStore: ViewStoreOf<SecurityFactors>) -> [SettingsRow<SecurityFactors>.Kind] {
+	func rows(viewStore: ViewStore<SecurityFactors.State, SecurityFactors.ViewAction>) -> [SettingsRow<SecurityFactors>.Kind] {
 		[
 			.header(L10n.SecurityFactors.subtitle),
-			.model(
-				title: L10n.SecurityFactors.SeedPhrases.title,
-				subtitle: L10n.SecurityFactors.SeedPhrases.subtitle,
-				detail: viewStore.seedPhrasesDetail,
-				hints: viewStore.seedPhraseHints,
-				icon: .asset(AssetResource.seedPhrases),
-				action: .seedPhrasesButtonTapped
-			),
-			.model(
-				title: L10n.SecurityFactors.LedgerWallet.title,
-				subtitle: L10n.SecurityFactors.LedgerWallet.subtitle,
-				detail: viewStore.ledgerWalletsDetail,
-				icon: .asset(AssetResource.ledger),
-				action: .ledgerWalletsButtonTapped
-			),
+			model(kind: .device, hints: viewStore.deviceHints),
+			.header(L10n.SecurityFactors.hardware),
+			model(kind: .arculusCard),
+			model(kind: .ledgerHqHardwareWallet),
+			.header(L10n.SecurityFactors.information),
+			model(kind: .password),
+			model(kind: .offDeviceMnemonic),
 		]
 	}
+
+	func model(kind: FactorSourceKind, hints: [Hint.ViewState] = []) -> SettingsRow<SecurityFactors>.Kind {
+		.model(
+			title: kind.title,
+			subtitle: kind.details,
+			hints: hints,
+			icon: .asset(kind.icon),
```

Comment:
`kind.icon` is an `ImageResource`, and not an `ImageAsset`. This is what triggered all the changes that you can see on the PR (basically use `ImageResource` instead of `ImageAsset` in `SettingsRow`). 
This is a good change anyway since we are using Swift's native type rather than a custom one from Swiftgen library